quinn-proto/src/config.rs

```diff
@@ -623,6 +624,12 @@ pub struct EndpointConfig {
     /// Create a cid generator for local cid in Endpoint struct
     pub(crate) connection_id_generator_factory:
         Arc<dyn Fn() -> Box<dyn ConnectionIdGenerator> + Send + Sync>,
+
+    /// Initial remote CID generator factory
```

Comment:
Would it be more correct/comprehensible to call this `dst` instead of `remote` everywhere? IIRC we mostly use `dst` for packets and `remote` for the connection state. Since these initial `dst_cid`s are never acknowledged by the remote peer, I think it would be more clear to refer to this as "destination CID" or `dst_cid` throughout.

@Ralith thoughts?

Reply:
That makes sense to me. Better matches RFC9000 terminology, too.

Reply:
Indeed, I was hesitated on its name.